packages/backend-defaults/src/entrypoints/database/databaseServiceFactory.ts

```diff
@@ -48,6 +50,12 @@ export const databaseServiceFactory = createServiceFactory({
             },
           }),
         );
+
+    rootLifecycle.addShutdownHook(async () => {
+      await databaseManager.shutdown({ logger: rootLogger });
```

Comment:
`.fromConfig` has an optional second options argument. You may want to pass in `{ rootLogger }` there as a new optional (to make it a nonbreaking change to `DatabaseManagerOptions`) field. Then the shutdown will have an optional logger to leverage already upfront.

On that same note, you could also pass in the `rootLifecycle` there, and have the database manager register a _private_ shutdown (keeping the external API unchanged) internally if passed in.